app/templates/components/event-card.hbs

```diff
@@ -9,7 +9,7 @@
       </div>
     {{/unless}}
   {{/if}}
-  <div style="height: 100%;"
+  <div style="{{if this.isWide 'height: auto;' 'height: 100%;'}}"
```

Comment:
use class instead of style, create classes in utils.scss `h-auto` & `h-100 `respectively

Reply:
@MeghalBisht yes ma'am